### DISCUSSION TITLE
Prefix strategy omits index.html in output folder, how to handle this?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt-modules/i18n#1764] [Q&A]
It seems obvious that index.html is not included in the build when the prefix stategy is selected, however I'm wondering how you make this work so that / redirects to /nl or /en, depending on the browser language? (I'm using Nginx, I suspect this might have something to do with this)

**Comment by ineshbose** [comment]
Hi, thank you for opening a discussion on this repository. It would be helpful to know what version of Nuxt and `@nuxtjs/i18n` you're using, along with your nginx configuration if possible please.

I would think that `/` onwards the entire app would be served (instead of specifying `/nl` and `/en` in your configuration if used), and the user would already be redirected to the browser locale. Keep in mind that there may be a current issue with v8 regarding detecting default locale - https://github.com/nuxt-modules/i18n/issues/1632
